src/components/views/rooms/MessageComposerInput.js

```diff
@@ -91,7 +74,7 @@ module.exports = React.createClass({
             // The textarea element to set text to.
             element: null,
 
-            init: function(element, roomId) {
+            init: function (element, roomId) {
```

Comment:
Not sure if this was intentional, but we should stick to the same style (ie. with no space)